src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMMOUNT = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private int supply;
+    private int buy;
+    private int result;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFile(fromFileName);
+        writeToFile(toFileName, stringBuilder(supply, buy, result));
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(toFileName, report);
```
lets refactor it like this
createReport - should report ready report in 1 String